docs/deteriorate.rst

```diff
@@ -68,8 +68,7 @@ Types
 
 :clothes:  All clothing pieces that have an armor rating of 0 and are lying on
            the ground.
-:corpses:  All non-dwarf corpses and body parts. This includes potentially
-           useful remains such as hair, wool, hooves, bones, and skulls. Use
-           them before you lose them!
+:corpses:  All resident corpses and body parts. To keep useable remains such as
+           hair, wool, hooves, bones, and skulls, specify --useable.
```

Comment:
since the name of the command is `deteriorate`, I would expect `deteriorate --useable` to cause useable items to rot away, but the opposite is indicated here.

I suggest instead calling the option `--keep-useable` and offering a short form of `-k` or `-u`. Also, you should probably additionally allow the alternate American spelling of `--keep_usable`.